docs/fundamentals/code-analysis/quality-rules/ca1854.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "CA1854: Prefer the 'IDictionary.TryGetValue(TKey, out TValue)' method"
+description: "Learn about code analyzer rule CA1854 - Prefer the 'IDictionary.TryGetValue(TKey, out TValue)' method"
+ms.date: 06/1/2022
+ms.topic: reference
+f1_keywords:
+    - CA1854
+helpviewer_keywords:
+    - CA1854
+author: CollinAlpert
+ms.author: CollinAlpert
```

Comment:
`ms.author` must be a Microsoft employee (it's also an internal alias, not GitHub username afaik).

```suggestion
ms.author: gewarren
```